pages/applications/add-case.tsx

```diff
@@ -99,6 +115,9 @@ export default function AddCasePage({ user }: PageProps): JSX.Element {
       handleSaveApplication={handleSaveApplication}
       ethnicity={ethnicity}
       setEthnicity={setEthnicity}
+      dataTestId="test-add-case-page"
+      isSaving={isSaving}
+      userError={userError}
```

Comment:
This component does not appear to have the `isSaving` and `userError` keys:
https://github.com/LBHackney-IT/lbh-housing-register/blob/e3b81f4691ec36c1e700b71a0d7ab33d79e7fc2b/components/admin/MainApplicantForm.tsx#L95-L106

Reply:
They are not in main yet, but are in this feature branch and also in development which this branch is based on. Are you perhaps looking at the component in main?

Reply:
You're right. I was looking into main, my bad.
Didn't notice we have a `development` branch on this repository.